glide-core/src/client/value_conversion.rs

```diff
@@ -26,6 +26,7 @@ pub(crate) enum ExpectedReturnType {
     ArrayOfMemberScorePairs,
     ZMPopReturnType,
     KeyWithMemberAndScore,
+    GeoSearchReturnType(bool, bool, bool),
```

Comment:
```suggestion
    GeoSearchReturnType(u8),
```
You don't perform targeted handling for extra data returned, but you calculate the number of `WITH*` - so you probably can store it here